src/AaveV2_Eth_ServiceProviders_20231907/TestTrade.s.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {Script} from 'forge-std/Script.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+
+import {IMilkman} from './interfaces/IMilkman.sol';
+
+contract TestTrade is Script {
+  address public constant MILKMAN = 0x11C76AD590ABDFFCD980afEC9ad951B160F02797;
+  address public constant CHAINLINK_PRICE_CHECKER = 0x4D2c3773E69cB69963bFd376e538eC754409ACFa;
+
+  // forge script src/AaveV2_Eth_ServiceProviders_20231907/TestTrade.s.sol:TestTrade --rpc-url mainnet --broadcast --private-key ${PRIVATE_KEY} --slow -vvvv
```

Comment:
Can we have this in NatSpec at the top of the contract with an explanation as to what the contract is? It makes sense to have, but since it tests "in prod" it can be a bit confusing as to how it works.